app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -71,7 +71,7 @@ def custom_index_relation(collection)
       param :http_proxy_id, :number, :desc => N_("ID of a HTTP Proxy")
       param :arch, String, :desc => N_("Architecture of content in the repository")
       param :retain_package_versions_count, :number, :desc => N_("The maximum number of versions of each package to keep.")
-
+      param :metadata_expire, :number, :desc => N_("time to expire yum metadata in seconds. Only relevant for custom yum repositories.")
```

Comment:
`param :metadata_expire, :number, :desc => N_("Time to expire yum metadata in seconds. Only relevant for custom yum repositories.")`